### DISCUSSION TITLE
Entrega de prácticas

### DISCUSSION THREAD

**Discussion by antoniocg97** [Launch-X-Latam/MisionFrontEnd#146] [Q&A]
Hola, como estan? tengo una duda... como se entregan las prácticas aquí en github?

**Comment by EsthebanDiego** [comment]
he visto que lo que se tiene que hacer es crear un repositorio, de tus prácticas, y subir en ese repositorio lo que se valla pidiendo

**Comment by ByJ4Xx** [comment]
En la sesión de hoy el MC dijo que luego (no especifico una fecha) abrira un formulario en el cual podemos subir nuestras practicas. Sin embargo, de mientras te recomiendo crear un repositorio en GitHub como dijo el comentario anterior, muchos compañeros lo han hecho asi y de paso te vas familiarizando con la plataforma

**Comment by antoniocg97** [comment]
ok, muchas gracias por sus respuestas, les agradezco
